<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoardPage</class>
 <widget class="QWidget" name="BoardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1058</width>
    <height>760</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QSplitter" name="hsplitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>5</number>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="editionWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="vsplitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="childrenCollapsible">
          <bool>false</bool>
         </property>
         <widget class="QTabWidget" name="tabWidget_2">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_4">
           <attribute name="title">
            <string>Pages</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="PagePickerWidget" name="pagePicker" native="true"/>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QTabWidget" name="tabWidget_3">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_5">
           <attribute name="title">
            <string>Elements</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="ElementPickerWidget" name="visuElementPicker" native="true"/>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QTabWidget" name="tabWidget_4">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="tab_6">
           <attribute name="title">
            <string>Data</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="ParameterPickerWidget" name="parameterPicker" native="true"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_7">
           <attribute name="title">
            <string>Properties</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="PropertiesPickerWidget" name="propertiesPicker" native="true"/>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayoutBoard">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ParameterPickerWidget</class>
   <extends>QWidget</extends>
   <header>ui/widgets/parameter_picker_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PagePickerWidget</class>
   <extends>QWidget</extends>
   <header>ui/widgets/page_picker_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ElementPickerWidget</class>
   <extends>QWidget</extends>
   <header>ui/widgets/element_picker_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PropertiesPickerWidget</class>
   <extends>QWidget</extends>
   <header>ui/widgets/properties_picker_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
